src/Shell.php

```diff
@@ -1201,13 +1223,14 @@ public function getLastExecSuccess(): bool
      */
     public function formatException(\Exception $e): string
     {
-        $message = $e->getMessage();
-        if (!$e instanceof PsyException) {
-            if ($message === '') {
-                $message = \get_class($e);
-            } else {
-                $message = \sprintf('%s with message \'%s\'', \get_class($e), $message);
-            }
+        if ($e instanceof BreakException) {
+            return sprintf("  <fg=white;bg=blue;options=bold> INFO </> %s.", $e->getRawMessage());
+        } elseif ($e instanceof PsyException) {
+            $message = \sprintf('%s in %s on line %d', $e->getRawMessage(), $e->getFile(), $e->getLine());
```

Comment:
we can probably often (always?) leave off `on line 1` for "in eval()d code" exceptions.

ideally we'd only do that if it was single line input, and couldn't possibly be on any other line, but if that's too hard to get at it's probably fine to just always leave it off for line === 1.

Reply:
I've leaved off when line === 1.